packages/base-dao/test/datamodel.graphql

```diff
@@ -0,0 +1,100 @@
+directive @unique on FIELD_DEFINITION | FIELD
```

Comment:
copy-pasting these files to all the packages might lead to maintenance problems, as the test schema changes, each package will have a slightly different schema which will be difficult to keep track of